commands/runserver/runserver.drush.inc

```diff
@@ -131,9 +131,12 @@ function drush_core_runserver($uri = NULL) {
     drush_start_browser($browse, 2);
   }
   // Start the server using 'php -S'.
-  if (drush_drupal_major_version() >=8) {
+  if (drush_drupal_major_version() >= 8) {
     $extra = ' "' . __DIR__ . '/d8-rs-router.php"';
   }
+  elseif (drush_drupal_major_version() == 7) {
+    $extra = ' "' . __DIR__ . '/d7-rs-router.php"';
+  }
```

Comment:
Given that the current code is already doing a conditional, this PR isn't making the situation any worse. I'd propose putting off improving the organization to a follow-up PR after fixing the bug.